src/mds/Migrator.h

```diff
@@ -174,6 +174,9 @@ class Migrator {
   void export_logged_finish(CDir *dir);
   void handle_export_notify_ack(const cref_t<MExportDirNotifyAck> &m);
   void export_finish(CDir *dir);
+  void encode_export_prep_trace(bufferlist& bl, CDir *bound, CDir *dir, export_state_t &es, 
+                               set<inodeno_t> &inodes_added, set<dirfrag_t> &dirfrags_added);
+  void decode_export_prep_trace(const bufferlist &bl, mds_rank_t oldauth, MDSContext::vec &finished);
```

Comment:
let's use the conventional `bufferlist::const_iterator& blp` instead of `const bufferlist& bl`.

Reply:
done